### PR TITLE
D training

### PR DESCRIPTION
**Pull request by dpatel77**
Split the dataset into train/val/test in 80/10/10 split

### CODE REVIEW

File: vision_project/vision_models/imageloader.py [hunk 1]
```diff
@@ -201,30 +254,75 @@ def create_dataset(self):
...
-        print("Dataset created, you can now iterate over the dataset")
+        print(f"Dataset for {split} created, you can now iterate over the dataset")
 
         return dataset
 
+    def analyze_split(self):
+        """
+        Analyze the split of the dataset.
+        """
+        self.label_df['split'] = pd.cut(
```
Comment (prashantkul) [inline vision_project/vision_models/imageloader.py]: Should this be self.label_coordinates_csv ? That is the file we're using to generate labels and features for Stage 1 model.

File: vision_project/vision_models/imageloader.py [hunk 2]
```diff
@@ -137,21 +158,53 @@ def _combine_labels(self, labels):
...
+
+        # train_dataset = dataset.take(train_size)
+        # val_dataset = dataset.skip(train_size).take(val_size)
+        # test_dataset = dataset.skip(train_size + val_size).take(test_size)
+
+        # return train_dataset, val_dataset, test_dataset
+
+        self._set_random_seed(self.random_seed)  # Set the random seed
+
+        unique_ids = self.label_df['study_id'].unique()
```
Comment (prashantkul) [inline vision_project/vision_models/imageloader.py:183]: Same here, this should be self.label_coordinates_csv. These two files are very different. See the count of lines in those files:

```
(vision) (vision) prashantkulkarni@vision-project-vm:/opt/dataset$ cat train.csv | wc -l
1976
(vision) (vision) prashantkulkarni@vision-project-vm:/opt/dataset$ cat train_label_coordinates.csv | wc -l
48693
```
